packages/@navikt/side-menu/src/StatefulSideMenu.tsx

```diff
@@ -2,12 +2,13 @@ import * as React from 'react';
 import SideMenu, { Link } from './index';
 
 interface StatefulSideMenuProps {
-    heading: string;
+    heading?: string;
```

Comment:
Dette gjør komponenten en del mer komplisert når man etterhvert trenger å utvide mer. Når skal `heading` vises og ikke? Det åpner også for at dagens `SideMenu` ikke trenger å ha en `heading`. Ikke at det er noen big deal nå, men det er en endring på noe som er utenfor scopet for denne utvidelsen. 

Trenger ikke gjøre noe med dette nå, men på sikt så bør nok komponenten splittes opp mer.